src/components/NavBar.js

```diff
@@ -17,9 +17,9 @@ import { useUsers } from '../utils/users'
 
 // hardcode some demo users
 const identities = {
-  'dfdadc39-7335-404d-af66-c77cf13a15f8': 'Euan',
-  'd64b8476-3c5f-4caf-af6f-9a0f1c51d19f': 'Kris',
-  '2bfaa552-d9a5-41e9-a6c3-5be62b4433c8': 'April'
+  'euang@acmecorp.com': 'Euan',
```

Comment:
will this work even before the keys change from the ugly Dex identifiers to the emails?

Reply:
yes, because i updated the code to match by ` user.properties?.email` if the key is not matched.